scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/service/ScheduleManager.java

```diff
@@ -639,4 +636,50 @@ private RecurringScheduleEntity saveNewRecurringSchedule(RecurringScheduleEntity
 		}
 		return savedScheduleEntity;
 	}
+
+	/**
+	 * Calls private helper methods to delete the schedules from the database and
+	 * calls ScalingJobManager to delete scaling action jobs.
+	 *
+	 * @param appId
+	 */
+	@Transactional
+	public void deleteSchedules(String appId) {
+	
+		// Get all the specific date schedules for the specifies application id and delete them.
+		List<SpecificDateScheduleEntity> specificDateSchedules = specificDateScheduleDao
+				.findAllSpecificDateSchedulesByAppId(appId);
+		for (SpecificDateScheduleEntity specificDateScheduleEntity : specificDateSchedules) {
+			// Delete the specific date schedule from database
+			deleteSpecificDateSchedule(specificDateScheduleEntity);
+
+			// Ask ScalingJobManager to delete scaling job 
+			scheduleJobManager.deleteSimpleJob(specificDateScheduleEntity);
+			
+		}
+
+		// Get all the recurring schedules for the specifies application id and delete them.
+		List<RecurringScheduleEntity> recurringSchedules = recurringScheduleDao.findAllRecurringSchedulesByAppId(appId);
+		for (RecurringScheduleEntity recurringScheduleEntity : recurringSchedules) {
+			// Delete the specific date schedule from database
+			deleteRecurringSchedule(recurringScheduleEntity);
+			// Ask ScalingJobManager to delete scaling job
+		}
+	}
+
+	private void deleteSpecificDateSchedule(SpecificDateScheduleEntity specificDateScheduleEntity) {
+		try {
+
+			specificDateScheduleDao.delete(specificDateScheduleEntity);
+		} catch (DatabaseValidationException dve) {
+			validationErrorResult.addErrorForDatabaseValidationException(dve, "database.error.delete.failed",
+					"app_id=" + specificDateScheduleEntity.getAppId());
+			throw new SchedulerInternalException("Database error", dve);
+		}
+	}
+
+	private void deleteRecurringSchedule(RecurringScheduleEntity recurringScheduleEntity) {
+
```

Comment:
If you are going to put empty methods for future use, at least have it throw an exception with some catch phrase so it is easy to determine what needs to be completed and have it fail if we attempt to use it.